apps/trading/components/page-query-container/page-query-container.tsx

```diff
@@ -7,19 +7,22 @@ import { AsyncRenderer } from '@vegaprotocol/ui-toolkit';
 interface PageQueryContainerProps<TData, TVariables> {
   query: DocumentNode;
   options?: QueryHookOptions<TData, TVariables>;
-  children: (data: TData) => ReactNode;
+  render: (data: TData) => ReactNode;
```

Comment:
IMHO render function as a child is an anti pattern. It's much more clear to use `render()` than `children()` and <Component render={() => <Child/>}/> than <Component>{() => <Child/>}</Component>